sources/csv/csv.go

```diff
@@ -18,6 +19,9 @@ func (t *table) start(reader *csv.Reader) {
 	defer close(t.rows)
 
 	headers, err := reader.Read()
+	if _, ok := err.(*csv.ParseError); ok {
+		log.Printf("csv.go received encoding/csv.ErrQuote, this can happen when a csv uses the wrong delimiter. Current delimiter is %q.", reader.Comma)
```

Comment:
Seems a bit specific to the use case you had. What about something like `error parsing CSV: extraneous ". This can happen when the CSV is malformed, or when the wrong delimiter is used`